examples/stable/mongodb/mongo-blueprint.yaml

```diff
@@ -55,7 +55,7 @@ actions:
           - pipefail
           - -c
           - |
-            host="{{ .Object.metadata.labels.release }}-mongodb.{{ .StatefulSet.Namespace }}.svc.cluster.local"
+            host='{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}-mongodb-0.{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}-mongodb-headless.{{ .StatefulSet.Namespace }}.svc.cluster.local'
```

Comment:
were we not able to connect to the running mongo using clusterIP service even after using the correct mongo client version.

Reply:
As per statefulset created from helm chart installation, the host would be resolved using the above name, since, mongodb-0(primary) and mongodb-1(secondary) are present in mongodb statefulset.